packages/terra-hookshot/tests/jest/HookshotDefaultExample.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-curly-brace-presence */
```

Comment:
?

Reply:
Removing the curly braces brings up the same error with no-unescaped-entities, and when removing the quotes leads to another `A control must be associated with a text label` error. I could fix this by adding an empty `aria-label` if that would be an acceptable change?

Reply:
The aria-label seems like an appropriate change.

Reply:
Added aria-label here: [d678858](https://github.com/cerner/terra-framework/pull/1166/commits/d67885871dee3d05aa83267625adaa0b9134ccc5).